docs/tutorials/services/source-ip.md

```diff
@@ -132,7 +132,7 @@ client_address=10.240.0.5
 client_address=10.240.0.3
 ```
 
-Note that these are not your IPs, they're cluster internal IPs. This is what happens:
+Note that these are not right client IPs, they're cluster internal IPs. This is what happens:
```

Comment:
Ok, cool. In that case, can you change the wording to: "Note that these are not the correct client IPs, they are the internal IPs used within the cluster." That should make it more clear. Thx